testing/src/test/java/io/stargate/it/limiter/GlobalRateLimitingTest.java

```diff
@@ -0,0 +1,46 @@
+package io.stargate.it.limiter;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import io.stargate.db.DbActivator;
+import io.stargate.it.BaseOsgiIntegrationTest;
+import io.stargate.it.driver.CqlSessionExtension;
+import io.stargate.it.driver.CqlSessionSpec;
+import io.stargate.it.storage.StargateParameters;
+import io.stargate.it.storage.StargateSpec;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@StargateSpec(parametersCustomizer = "buildParameters")
+@ExtendWith(CqlSessionExtension.class)
+@CqlSessionSpec(
+    initQueries = {"CREATE TABLE IF NOT EXISTS test (k text, v int, PRIMARY KEY(k, v))"})
+public class GlobalRateLimitingTest extends BaseOsgiIntegrationTest {
+
+  @SuppressWarnings("unused") // referenced in @StargateSpec
+  public static void buildParameters(StargateParameters.Builder builder) {
+    builder.putSystemProperties(DbActivator.RATE_LIMITING_ID_PROPERTY, "GlobalRateLimiting");
+    builder.putSystemProperties("stargate.limiter.global.rate_qps", "1");
+  }
+
+  @Test
+  @DisplayName("Should rate limit queries")
+  public void testRateLimited(CqlSession session) {
+    int QUERIES = 5;
+    // We're executing a point query on an empty table 5 times, so this should be really fast, even
```

Comment:
Shouldn't we provide a unit test for rate limiter? I think that unit test allows to test this behavior better (with isolation and without external factors)